### PR TITLE
Fix kube jobs generator's bugs.

### PR DESCRIPTION
**Pull request by gongweibao**

### CODE REVIEW

File: benchmark/fluid/kube_gen_job.py [hunk 1]
```diff
@@ -108,10 +109,10 @@ def gen_job():
     tn_container["ports"][0]["containerPort"] = spreadport
 
     envs.append({"name": "PADDLE_JOB_NAME", "value": args.jobname})
-    envs.append({"name": "TRAINERS", "value": str(args.trainers)})
-    envs.append({"name": "PSERVERS", "value": str(args.pservers)})
+    envs.append({"name": "PADDLE_TRAINERS", "value": str(args.trainers)})
+    envs.append({"name": "PADDLE_PSERVERS", "value": str(args.pservers)})
     envs.append({"name": "ENTRY", "value": args.entry})
-    envs.append({"name": "PADDLE_INIT_PORT", "value": str(args.port)})
+    #envs.append({"name": "PADDLE_INIT_PORT", "value": str(args.port)})
```
Comment (Yancey0623) [inline benchmark/fluid/kube_gen_job.py]: Delete the comment code.

File: benchmark/fluid/Dockerfile [hunk 2]
```diff
@@ -1,22 +1,30 @@
...
+
+RUN pip install -U pip
+RUN pip install -U kubernetes paddlepaddle
 
 RUN sh -c 'echo "import paddle.v2 as paddle\npaddle.dataset.cifar.train10()\npaddle.dataset.flowers.fetch()" | python'
 RUN sh -c 'echo "import paddle.v2 as paddle\npaddle.dataset.mnist.train()\npaddle.dataset.mnist.test()\npaddle.dataset.imdb.fetch()" | python'
 RUN sh -c 'echo "import paddle.v2 as paddle\npaddle.dataset.imikolov.fetch()" | python'
 RUN pip uninstall -y paddlepaddle && mkdir /workspace
 
+ADD *.whl /
```
Comment (Yancey0623) [inline benchmark/fluid/Dockerfile]: Don't move `ADD *.whl /`? We can cache `ADD https://raw.githubusercontent.com/PaddlePaddle/cloud/develop/docker/paddle_k8s /usr/bin ` .

Comment (gongweibao) [reply benchmark/fluid/Dockerfile]: Done.

File: benchmark/fluid/kube_gen_job.py (hunk 1 above)
Comment (gongweibao) [reply benchmark/fluid/kube_gen_job.py]: Done.

File: benchmark/fluid/Dockerfile [hunk 3]
```diff
@@ -1,11 +1,17 @@
 FROM nvidia/cuda:9.0-cudnn7-devel-ubuntu16.04
+
+ARG UBUNTU_MIRROR
```
Comment (typhoonzero) [inline benchmark/fluid/Dockerfile]: Not all versions of docker support `ARG`. And change one line of Dockerfile seems easier?

**Review by typhoonzero — COMMENTED** [review]
I see some duplicate changes in another PR, please merge that and update this PR later.

File: benchmark/fluid/Dockerfile (hunk 3 above)
Comment (gongweibao) [reply benchmark/fluid/Dockerfile]: Done.

**Review by Yancey0623 — APPROVED** [review]
LGTM

**Pull request merged**